makefile

```diff
@@ -0,0 +1,19 @@
+.DEFAULT_GOAL := all
+.PHONY: all build stage deploy-release
+
+TODAY ?=$(shell date +%Y-%m-%d)
+VERSION=v$(TODAY)
+
+all: build
+build:
+	python -m sssom_on_fhir --convert-from-content-dir
+
+stage:
+	python -m sssom_on_fhir --stage-release
+
+release/:
+	mkdir -p $@
+
+deploy-release: stage | release/
```

Comment:
## Add command: `deploy-release`
This will create a new GitHub release after a build has been run, which can be done via `make build` or running the Python command.